editor/styling/stylesheets/_dialogs.scss

```diff
@@ -6,7 +6,7 @@ $spacing-largest: 40px;
 
 @use "sass:string";
 @mixin font-size-default { -fx-font-size: 110%; }
-@mixin font-size-header { -fx-font-size: 170%; }
+@mixin font-size-header { -fx-font-size: 22px; }
```

Comment:
170% was almost 22px, but not quite. I made headers consistent across dialogs, markdown, and now editor scripts.